File: 프로그래머스/0/181865. 간단한 식 계산하기/간단한 식 계산하기.js
function solution(binomial) {
  const cal = binomial.split(' ')
  switch (cal[1]) {
    case '+': return +cal[0] + +cal[2]
    case '-': return +cal[0] - +cal[2]
    case '*': return +cal[0] * +cal[2]
  }
} 

File: 프로그래머스/0/181865. 간단한 식 계산하기/간단한 식 계산하기.java
class Solution {
    public int solution(String binomial) {
        String[] cal = binomial.split(" ");
        
        switch (cal[1]) {
            case "+":
                return Integer.parseInt(cal[0]) + Integer.parseInt(cal[2]);
            case "-":
                return Integer.parseInt(cal[0]) - Integer.parseInt(cal[2]);
            case "*":
                return Integer.parseInt(cal[0]) * Integer.parseInt(cal[2]);
        }
        return 0;
    }
}
